tests/unittests/Starboard/CommonCryptoTests.mm

```diff
@@ -408,75 +402,170 @@ void _encryptDecryptSameAsOriginal(CCAlgorithm algorithm,
-                            ::testing::make_tuple(kCCAlgorithm3DES, kCCEncrypt, 0, kCCKeySize3DES, kCCBlockSize3DES - 1, kCCBlockSize3DES + 1, kCCBlockSize3DES * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithm3DES, kCCEncrypt, kCCOptionECBMode, kCCKeySize3DES, kCCBlockSize3DES, kCCBlockSize3DES, kCCBlockSize3DES * 2, nullptr),
-                            ::testing::make_tuple(kCCAlgorithm3DES, kCCEncrypt, kCCOptionPKCS7Padding, kCCKeySize3DES, kCCBlockSize3DES - 1, kCCBlockSize3DES, kCCBlockSize3DES * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithm3DES, kCCEncrypt, kCCOptionECBMode | kCCOptionPKCS7Padding, kCCKeySize3DES, kCCBlockSize3DES, kCCBlockSize3DES + 1, kCCBlockSize3DES * 3, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmAES128, kCCEncrypt, 0, kCCKeySizeAES128, kCCBlockSizeAES128, kCCBlockSizeAES128, kCCBlockSizeAES128 * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithmAES128, kCCEncrypt, 0, kCCKeySizeAES128, kCCBlockSizeAES128 - 1, kCCBlockSizeAES128 + 1, kCCBlockSizeAES128 * 2, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmAES128, kCCEncrypt, kCCOptionECBMode, kCCKeySizeAES128, kCCBlockSizeAES128, kCCBlockSizeAES128, kCCBlockSizeAES128 * 2, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmAES128, kCCEncrypt, kCCOptionPKCS7Padding, kCCKeySizeAES128, kCCBlockSizeAES128 - 1, kCCBlockSizeAES128, kCCBlockSizeAES128 * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithmAES128, kCCEncrypt, kCCOptionECBMode | kCCOptionPKCS7Padding, kCCKeySizeAES128, kCCBlockSizeAES128, kCCBlockSizeAES128 + 1, kCCBlockSizeAES128 * 3, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmRC2, kCCEncrypt, 0, kCCKeySizeMinRC2, kCCBlockSizeRC2, kCCBlockSizeRC2, kCCBlockSizeRC2 * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithmRC2, kCCEncrypt, 0, kCCKeySizeMaxRC2, kCCBlockSizeRC2 - 1, kCCBlockSizeRC2 + 1, kCCBlockSizeRC2 * 2, secret2),
-                            ::testing::make_tuple(kCCAlgorithmRC2, kCCEncrypt, kCCOptionECBMode, kCCKeySizeMinRC2, kCCBlockSizeRC2, kCCBlockSizeRC2, kCCBlockSizeRC2 * 2, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmRC2, kCCEncrypt, kCCOptionPKCS7Padding, kCCKeySizeMaxRC2, kCCBlockSizeRC2 - 1, kCCBlockSizeRC2, kCCBlockSizeRC2 * 2, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmRC2, kCCEncrypt, kCCOptionECBMode | kCCOptionPKCS7Padding, kCCKeySizeMinRC2, kCCBlockSizeRC2, kCCBlockSizeRC2 + 1, kCCBlockSizeRC2 * 3, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmRC4, kCCEncrypt, 0, kCCKeySizeMinRC4, 2, 5, 7, nullptr),
-                            ::testing::make_tuple(kCCAlgorithmRC4, kCCEncrypt, 0, kCCKeySizeMaxRC4, 0, 14, 14, nullptr)));
+INSTANTIATE_TEST_CASE_P(
```

Comment:
I do too, I'll turn clang off just for this block

---
In reply to: [70348637](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70348637) [](ancestors = 70348637)